thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/constant/AnomalyFeedbackType.java

```diff
@@ -1,5 +1,5 @@
 package com.linkedin.thirdeye.constant;
 
 public enum AnomalyFeedbackType {
-  ANOMALY, NOT_ANOMALY, ANOMALY_NO_ACTION, NO_FEEDBACK
+  ANOMALY, NOT_ANOMALY, ANOMALY_NEW_TREND, NO_FEEDBACK
```

Comment:
No. I can see it being used a lot in anomaly detection library. Also instead of having null as default, it's better to have a proper value.